charts/tidb-lightning/values.yaml

```diff
@@ -47,9 +47,13 @@ resources: {}
 
 nodeSelector: {}
 
+annotations: {}
+
 tolerations: []
 affinity: {}
 
+backend: importer # importer | tidb
```

Comment:
Which tidb version supports specifying the backend?

Reply:
It was tested with 3.0.3.